package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "eslint-plugin-wingsui-rules",
```

Comment:
we alse need to come up with a consistent package naming scheme. 
`@harnessui/core`, `@harness/css-types-loader` and now this, all follow different schemes. let's follow one!